regimes/at/identitites.go

```diff
@@ -0,0 +1,82 @@
+package at
+
+import (
+	"errors"
+	"regexp"
+	"strconv"
+
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/validation"
+)
+
+const (
+	// IdentityKeyTaxNumber represents the Austrian tax number (Steuernummer) issued to
+	// people that can be included on invoices inside Austria. For international
+	// sales, the registered VAT number (Umsatzsteueridentifikationsnummer) should
+	// be used instead.
+	IdentityKeyTaxNumber cbc.Key = "at-tax-number"
```

Comment:
I'm thinking that these should be Identity types also, in this case ChatGTP suggests `STNR`. The same could also be applied to Germany (but we don't want to do that yet!)